### DISCUSSION TITLE
Use hotkey to cycle through audio output sources

### DISCUSSION THREAD

**Discussion by ghost** [File-New-Project/EarTrumpet#1081] [Ideas]
I love that I'm able to right click on the taskbar icon to choose my audio source, but if I could apply this to a hotkey it would be beneficial when I have a fullscreen application open such as a video game. Is there a way to apply this that I'm not aware of?

edit: using the dev build and setting conditionals enabled me to do this - https://eartrumpet.app/

**Comment by Memphizzz** [comment]
I would love to see this too! Exactly for when you're gaming as mentioned above this would a great addition.

**Reply by ghost** [reply]
I was able to get it working using @healthazard's link #1012 . Let me know if you need help getting it working!

**Comment by healthazard** [comment]
I've also requested this, there is an alternative in dev build, tho - more: https://github.com/File-New-Project/EarTrumpet/discussions/1012

**Reply by ghost** [reply]
Thanks a lot for the link!!
